addons/bitcoin/Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "txtx-addon-network-bitcoin"
 version = "0.1.0"
 edition = "2021"
+license = "GPL-3.0"
```

Comment:
I used a GPL license for Stacks because we're pulling some GPL3 dependencies in that addon.
If this is not the case here, we should default to Apache 2!

Reply:
Done, thanks!